cli_tools/gce_vm_image_import/importer/disk_inspection_processor.go

```diff
@@ -0,0 +1,114 @@
+//  Copyright 2020 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package importer
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/common/disk"
+	"github.com/GoogleCloudPlatform/compute-image-tools/daisy"
+	daisyCompute "github.com/GoogleCloudPlatform/compute-image-tools/daisy/compute"
+	"google.golang.org/api/compute/v1"
+)
+
+type diskInspectionProcessor struct {
+	args              ImportArguments
+	computeDiskClient daisyCompute.Client
+	diskInspector     disk.Inspector
+}
+
+func (d *diskInspectionProcessor) process(pd persistentDisk) (persistentDisk, error) {
```

Comment:
I agree here with Eric that inspection and processing should be separate steps. Inspection should only do inspection while uefi disk processor could perform disk creation with UEFI_COMPATIBLE guest OS feature attached. Inspection would be a separate step after inflation (unless there are technical issues), while UEFI processor be in the list of processor that run on the disk.

Reply:
Moved the logic to the proper place as suggested, thanks!

Reply:
Thanks Tony! Took a look, and the current version that's uploaded still has the UEFI work inside bootable_processor, rather than having a list of processors.

Reply:
Split the logic to 3 processors now.